File: 1-two-sum/two-sum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        Dictionary<int,int> hashMap=new Dictionary<int,int>();

        for(int i=0;i<nums.Length;i++)
        {
            var sum=target-nums[i];

            if(hashMap.ContainsKey(sum))
            {
                return new[] {i,hashMap[sum]};
            }
            if(!hashMap.ContainsKey(nums[i]))
                hashMap.Add(nums[i],i);
        }
        return null;
    }
}

File: 1-two-sum/two-sum.ts
function twoSum(nums: number[], target: number): number[] {
    
    for(let i=0;i<nums.length;i++)
    {
        const numToFind:number=target-nums[i];
        
        for(let j=i+1;j<=nums.length;j++)
        {
            if(numToFind===nums[j])
            {
                return [i,j];
            }
        }
    }
};